src/vue/modules/issuance-explorer/components/issuances-table.vue

```diff
@@ -38,6 +42,46 @@
                 {{ issuance.asset }}
               </td>
 
+              <td
```

Comment:
IMO, request state logic takes up a lot of space here. What about extracting it to a separate component, called `request-state-viewer` (smth like [this](https://github.com/tokend/web-client/blob/2753e11f7782c86004f55a19ac5449690d221871/src/vue/modules/requests/shared/components/request-state-viewer.vue))?